LICENSE.md

```diff
@@ -1,4 +1,27 @@
-The Tokenize.jl package is licensed under the MIT "Expat" License:
+The JuliaSyntax.jl package is licensed under the MIT "Expat" License:
+
+> Copyright (c) 2021 Julia Computing and contributors
```

Comment:
Wouldn't it be easier to just add @KristofferC here on this line? I generally think it is good if LICENSE files just have the template language and no custom additions, makes it easier for users. Plus, in this case I doubt that legally it makes any difference.

I _do_ think it would be nice to ack @KristofferC explicitly, not for legal reasons but just for history, but maybe the README is a better place for that?

Reply:
Sure it'd be cleaner to consolidate.  But also I don't care too much about how clean the license file is. Whatever makes people happy.

Tokenize.jl is currently linked to from the README.

Reply:
I personally don't care too much about what is in the LICENSE.

Reply:
In that case, how about this: I reverted to the original license text for JuliaSyntax.jl. That was somewhat author-neutral and It's all MIT anyway.

git has the real history.

Reply:
Hmm I didn't feel quite right about this so I've gone back to including the Tokenize license how I had it. Will merge shortly; we can always simplify later if desired.